common/src/main/java/com/microsoft/identity/common/CodeMarker.java

```diff
@@ -0,0 +1,47 @@
+package com.microsoft.identity.common;
+
+public class CodeMarker {
+    private int marker;
+    private long timeMilliseconds;
```

Comment:
```suggestion
    private long timeInMilliSeconds;
```

Reply:
Can we also add a description of what this time represents?

Reply:
Done. Please let me know if the comment is able to describe the variable